content/security/gift.adoc

```diff
@@ -1,65 +0,0 @@
----
```

Comment:
Time to figure out how to purge the site?

https://www.jenkins.io/security/gift/

Even the search still indexes it…

Reply:
I've ran `curl -X PURGE https://www.jenkins.io/security/gift` and a complete purge on Fastly but the page is still present 🤷

Reply:
Yep, we only add, never delete.

Reply:
Issue opened: https://github.com/jenkins-infra/helpdesk/issues/3360

Reply:
> Yep, we only add, never delete.

Do you happen to know the reason for that?

Reply:
My guess (!) is that we just upload using the Azure CLI and never delete what's there.